<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DAB_MainWindow</class>
 <widget class="QMainWindow" name="DAB_MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>608</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DAB_MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBoxOperations">
    <property name="geometry">
     <rect>
      <x>69</x>
      <y>229</y>
      <width>461</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Opérations envoyées à la banque</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>42</y>
       <width>82</width>
       <height>114</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="radioButtonSolde">
        <property name="text">
         <string>Solde</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonRetrait">
        <property name="text">
         <string>Retrait</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonDepot">
        <property name="text">
         <string>Dépot</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonEnvoi">
        <property name="text">
         <string>Envoi</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>271</width>
       <height>58</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButtonNumeroDeCompte">
        <property name="text">
         <string>Numéro de compte</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditCompte"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelMontant">
        <property name="text">
         <string>Montant :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditMontant"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>59</x>
      <y>19</y>
      <width>211</width>
      <height>58</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="labelAdresse">
       <property name="text">
        <string>Adresse :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditAdresse"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelPort">
       <property name="text">
        <string>Port :</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditPort"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButtonConnexion">
    <property name="geometry">
     <rect>
      <x>129</x>
      <y>99</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Connexion</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_4">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>160</y>
      <width>471</width>
      <height>95</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="labelMessage">
       <property name="text">
        <string>Message de la banque :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="textEditMessage"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_5">
    <property name="geometry">
     <rect>
      <x>289</x>
      <y>9</y>
      <width>251</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="labelEtat">
       <property name="text">
        <string>Etat de la connexion</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="listWidgetEtat"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>608</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
